indico/modules/rb/client/js/components/SidebarMenu.jsx

```diff
@@ -103,35 +108,96 @@ function SidebarMenu({
         }
     ].filter(({onlyIf}) => onlyIf === undefined || onlyIf);
 
+    const [contactVisible, setContactVisible] = useState(false);
+    const [termsVisible, setTermsVisible] = useState(false);
+
     return (
-        <Sidebar as={Menu}
-                 animation="overlay"
-                 icon="labeled"
-                 vertical
-                 width="thin"
-                 direction="right"
-                 onHide={onClickOption}
-                 inverted
-                 visible={visible}
-                 styleName="sidebar">
-            {options.map(({key, text, icon, onClick, iconColor, active, tooltip}) => {
-                const item = (
-                    <Menu.Item as="a" key={key} active={active} onClick={() => {
-                        onClick();
-                        if (onClickOption) {
-                            onClickOption();
-                        }
-                    }}>
-                        <Icon name={icon} color={iconColor} />
-                        {text}
-                    </Menu.Item>
-                );
-                if (!tooltip) {
-                    return item;
-                }
-                return <Popup trigger={item} content={tooltip} key={key} position="left center" />;
-            })}
-        </Sidebar>
+        <>
+            <Sidebar as={Menu}
+                     animation="overlay"
+                     icon="labeled"
+                     vertical
+                     width="thin"
+                     direction="right"
+                     onHide={onClickOption}
+                     inverted
+                     visible={visible}
+                     styleName="sidebar">
+                {options.map(({key, text, icon, onClick, iconColor, active, tooltip}) => {
+                    const item = (
+                        <Menu.Item as="a" key={key} active={active} onClick={() => {
+                            onClick();
+                            if (onClickOption) {
+                                onClickOption();
+                            }
+                        }}>
+                            <Icon name={icon} color={iconColor} />
+                            {text}
+                        </Menu.Item>
+                    );
+                    if (!tooltip) {
+                        return item;
+                    }
+                    return <Popup trigger={item} content={tooltip} key={key} position="left center" />;
+                })}
+                <div styleName="bottom-align">
+                    {helpURL && (
+                        <Menu.Item as="a" onClick={() => {
+                            window.location.href = helpURL;
```

Comment:
can't we set a normal href? Using JS to change location is pretty ugly since you can't middle-click for example...